pkgs/tools/filesystems/hdl-dump/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  version = "2020-02-05";
+  pname = "hdl-dump";
+
+  src = fetchFromGitHub {
+    owner = "AKuHAK";
+    repo  = "hdl-dump";
+    rev = "b0d74679eaeb404cdfd5c43ec8fd18d9e2bbb038";
+    sha256 = "1k6m4px14cr0i8f4fpk4f1wp0sk4isbnbhib1vl7r4yg7564y1j1";
+  };
+
+  buildPhase = ''
+    make DEBUG=no
```

Comment:
A custom `buildPhase` isn't required for this, rather you can just use `makeFlags = [ "DEBUG=no" ];`.